### ISSUE TITLE
mktmpio fails on mac

### ISSUE THREAD

**Issue by rokka-n** [mktmpio/cli#1]
Create a db via curl, as mention in the guide on website.
Then on mac:

```
mktmpio --version
mktmpio version 0.7.0

mktmpio ls
Error listing instance: json: cannot unmarshal object into Go value of type []mktmpio.Instance
```

Creating db also fails with different errors. 
How to compile this, btw?

**Comment by rmg** [comment]
@rokka-n does `mktmpio config get` show your token? (don't paste it here if it does)

**Comment by rmg** [comment]
It looks like some instructions were missing from https://mktmp.io/get-started

If you go to that page now it should tell your API key and how to set it locally so that the `mktmpio` CLI will work properly:
- `mktmpio config set token <your token here>`

**Comment by rokka-n** [comment]
It does show the config with correct key

```
mktmpio config get
token: xxxx
url: https://mktmp.io/api/v1
```

**Comment by rmg** [comment]
Hm.. this is very strange! Are you behind a proxy by chance?

Could you try this?

```
curl -v -H "X-Auth-Token: <YOUR TOKEN>" -H "Accept: application/json" https://mktmp.io/api/v1/i
```

**Comment by rokka-n** [comment]
curl works just fine, it's something wrong with the binary 

```
[{"type":"redis","owner":"xxx","id":"6e585b0b23a7","host":"i.mktmp.io","age":10026107,"port":32888,"password":"xxx","remoteShell":{"cmd":["redis-cli","-h","i.mktmp.io","-p","32888","-a","xxx"]},"containerShell":["redis-cli","-a","xxx"]}]
```

**Comment by rmg** [comment]
@rokka-n can you give me output of `uname -rsm`?

**Comment by rokka-n** [comment]
```
uname  -rsm
Darwin 15.2.0 x86_64
```

**Comment by rmg** [comment]
@rokka-n I've got a Mac with `15.3.0` and one with `14.5.0` and I haven't been able to reproduce this using the published v0.7.0 binary on either of them. Surprisingly, even the 32-bit binary worked for me. 

I'm going to see if I can add some additional logging and possibly releasing it as a patch release.

Let me know if you have any luck building the binary yourself.

**Comment by rokka-n** [comment]
Ok, thanks.
What's the commands to build it?

**Comment by rmg** [comment]
If you have homebrew, I think this should get you up and running:
1. `brew install go`
2. `export GOPATH=~/go`
3. `go get github.com/mktmpio/cli`
4. `cd $GOPATH/github.com/mktmpio/cli`
5. `make`

**Comment by rmg** [comment]
@rokka-n I've done a bit of cleanup and added a `--debug` flag and released [v0.8.0](https://github.com/mktmpio/cli/releases/tag/0.8.0). Try the new version. To turn on debug just add `-d` or `--debug` as a flag _before_ the command (`mktmpio -d ls` not `mktmpio ls -d`).

**Comment by rokka-n** [comment]
Thanks, @rmg 
I finally manage to try in on another macbook and everything compiles and runs just fine.
